js/src/personal/amanda/Amanda.page.tsx

```diff
@@ -1,34 +1,45 @@
-import { Title, Text } from '@mantine/core'
+import { Button, Center, Title } from '@mantine/core'
+import { useViewportSize } from '@mantine/hooks'
+import { useState } from 'react'
 import styles from './Amanda.module.css'
+import { AmandaMainPage } from './Amanda.main.page.tsx'
 
 export function AmandaPage() {
+  const { height, width } = useViewportSize()
+  const [landingPage, setLandingPage] = useState(1)
+
+  // render loading page
+  if (landingPage === 1) {
```

Comment:
You mix landing + loading in strings
Might make sense to make this another component as well.